src/test/java/uk/gov/justice/digital/hmpps/keyworker/services/health/HealthCheckIntegrationTest.kt

```diff
@@ -0,0 +1,185 @@
+package uk.gov.justice.digital.hmpps.keyworker.services.health
+
+import com.amazonaws.services.sqs.model.GetQueueAttributesRequest
+import com.amazonaws.services.sqs.model.GetQueueAttributesResult
+import com.amazonaws.services.sqs.model.QueueAttributeName
+import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.get
+import com.nhaarman.mockito_kotlin.whenever
+import net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.test.util.ReflectionTestUtils
+import uk.gov.justice.digital.hmpps.keyworker.services.health.QueueAttributes.*
+
+class HealthCheckIntegrationTest : IntegrationTest() {
+
+  @Autowired
+  private lateinit var queueHealth: QueueHealth
+
+  @Autowired
+  @Value("\${sqs.queue.name}")
+  @Suppress("SpringJavaInjectionPointsAutowiringInspection")
```

Comment:
Nice, I wondered how to get rid of those stupid warnings

Reply:
intellij provides this as a suggestion on the warning